roles/keystone/templates/etc/keystone/uwsgi/keystone-main.ini

```diff
@@ -13,7 +13,7 @@ logfile-chmod = 644
 {% if keystone.uwsgi.method == 'socket' %}
 socket = /run/uwsgi/keystone-main.socket
 {% else %}
-http-socket = 127.0.0.1:{{ keystone.uwsgi.http_port.public }}
+socket = 127.0.0.1:{{ keystone.uwsgi.http_port.public }}
```

Comment:
I'll go check if it works, I think it would be nice to know what type of socket is being used.

I think we should use the unix socket whenever we can because there is less overhead due to TCP when compared to the http socket. I would like to to leave it in there as in the future. When Apache is at least  2.4.9 (on Ubuntu or RHEL), we will be able to use unix sockets with the proxy_uwsgi module.